extensions/lifecycle/lifecycleConsumer/task.js

```diff
@@ -30,6 +33,15 @@ function initAndStart() {
             setTimeout(initAndStart, 5000);
             return;
         }
+        healthServer.onReadyCheck(log => {
+            if (lifecycleConsumer.isReady()) {
+                return true;
+            }
+            log.error('LifecycleConductor is not ready!');
+            return false;
+        });
+        log.info('Starting HealthProbe server');
+        healthServer.start();
         log.info('management init done');
```

Comment:
You can move that log line before `healthServer.onReadyCheck()` since it relates to the management layer initialization.